### PR TITLE
Added a corner case to the test cases for Calculator app

### PR DESCRIPTION
**Pull request by KarthickPN**
From [program2](https://github.com/fsareshwala/cs404.1/blob/master/program2.md),

> Repeat until operator stack is empty or _token at top of operator stack has lower precedence than input operator_.   

Adding a test case for the above italicized section.

### CODE REVIEW

File: java/test/edu/berkeley/cs/app/Calculator_T.java [hunk 1]
```diff
@@ -17,6 +17,7 @@ public void testSimpleArithmetic() {
     Assert.assertEquals(-1, Calculator.evaluate("3 - 4"), epsilon);
     Assert.assertEquals(12, Calculator.evaluate("3 * 4"), epsilon);
     Assert.assertEquals(3.75, Calculator.evaluate("15 / 4"), epsilon);
+	Assert.assertEquals(-3, Calculator.evaluate("2 - 5 * 5 / 5"), epsilon);
```
Comment (fsareshwala) [inline java/test/edu/berkeley/cs/app/Calculator_T.java:20]: I would actually make this a separate test since it is testing operator precedence.

Comment (fsareshwala) [reply java/test/edu/berkeley/cs/app/Calculator_T.java:20]: nit: please run `tools/format.sh` to ensure proper format in the repository

Comment (KarthickPN) [reply java/test/edu/berkeley/cs/app/Calculator_T.java]: Done.
